artifact/src/kind.rs

```diff
@@ -133,6 +133,10 @@ impl ArtifactKind {
     /// Derived from [`KnownArtifactKind::Trampoline`].
     pub const TRAMPOLINE_PHASE_2: Self =
         Self::from_static("trampoline_phase_2");
+
+    /// Manifest identifier
+    pub const MEASUREMENT_CORPUS: Self =
+        Self::from_static("measurement_corpus");
```

Comment:
This shouldn't be necessary if my assumptions above are accurate, since we can use the KnownArtifactKind we're defining.

Reply:
Yeah I think this came from some earlier attempts